cirq-core/cirq/sim/sparse_simulator_test.py

```diff
@@ -1270,6 +1270,14 @@ def test_nondeterministic_mixture_noise():
     assert result1 != result2
 
 
-def test_unsupported_noise_fails():
-    with pytest.raises(ValueError, match='noise'):
-        cirq.Simulator(noise=cirq.amplitude_damp(0.5))
+def test_noise_model():
+    q = cirq.LineQubit(0)
+    circuit = cirq.Circuit(cirq.I(q), cirq.measure(q))
```

Comment:
There is a class of bugs that make `sum(result.measurements['0'])[0]` equal to zero exactly (e.g. one such bug is the simulator accepting but ignoring the noise). These bugs would pass undetected by the test.

Suggestion: Replace identity with `cirq.H(q)` and change the assert to something like

```python
assert 40 <= sum(result.measurements['0'])[0] < 60
```

Feel free to use tighter bounds, I just made those up. They should be gapped away from zero on the left and away from 100 on the right, though.

Reply:
thanks for the suggestion, I just updated it!